### ISSUE TITLE
Bug: after upgrade to v1.5.1.RELEASE in CartBaseController.populateEligibleForOneClickCheckout throws AccessDeniedException

### ISSUE THREAD

**Issue by tttodorov13** [nemesis-software/nemesis-support#10]
After upgrade to v1.5.1.RELEASE in `CartBaseController.populateEligibleForOneClickCheckout ` `customerFacade.getDefaultAddress()` and `customerFacade.getDefaultPaymentInfo()` throw `AccessDeniedException`.

**Comment by ptahchiev** [comment]
Hello Toshko,

can you please provide some more details: 
 - What specifically you click on the frontend?
 - Have you extended any of the services from the Nemesis Platform?
 - Do you have the `personalization` module in your classpath and have you modified the `personalization.csv`?

**Comment by tttodorov13** [comment]
Hello Petar,
- I am trying to access /cart URL from the root of the application. Right now I have developed workaround checking by userFacade.isCurrentUserAnonymous().
- No service was extended by me observing the bug.
- The encoding is the only change made in personalization.csv since Initial commit.

**Comment by ptahchiev** [comment]
Hello Toshko,

thank you for your message. I was able to reproduce the issue. Working on a fix, which will be included in a 1.5.2 this week. Stay tuned.

**Comment by ptahchiev** [comment]
Hi Toshko,
the changes need to be in the `CartBaseController`. Please change your `populateEligibleForOneClickCheckout` method to the following:

````
    protected void populateEligibleForOneClickCheckout(final Model model) {
        model.addAttribute("oneClickCheckoutEligible", !getUserFacade().isCurrentUserAnonymous() && customerFacade.getDefaultAddress() != null
                        && customerFacade.getDefaultPaymentInfo() != null);

    }
````
